### PR TITLE
Add radius values to theme

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Add a scale of radius values to theme

## Justify why they are needed
Consistency

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #886** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/886" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/886?utm_source=stack-comment).

**Review by KarnellSchultz — APPROVED** [review]
loop-nft-gif-by-xponentialdesign

File: packages/ui/src/lib/theme/radius.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+type RadiusAlias = 'xs' | 'sm' | 'md' | 'lg' | 'xl'
+
+export const radius: Partial<Record<number | RadiusAlias, string>> = {
```
Comment (alebedev) [inline packages/ui/src/lib/theme/radius.ts:3]: Do you think we'd ever need numeric indexes?  Names look better and are probably easier to use

**Review by alebedev — APPROVED** [review]
image

**Pull request merged**